lib/utils.pm

```diff
@@ -1656,4 +1658,35 @@ sub ensure_ca_certificates_suse_installed {
     }
 }
 
+sub service_start {
```

Comment:
It means that we have two function across code base wit quite similar name `start_service` vs `service_start`. It is little bit confusing, isn't it?

Reply:
yes, how about I change them to common_service_start and common_sevice_status?

Reply:
I've changed them to prefixed with common_, thanks